kafka/producer/base.py

```diff
@@ -111,12 +160,16 @@ def __init__(self, client, async=False,
                  codec=None,
                  batch_send=False,
                  batch_send_every_n=BATCH_SEND_MSG_COUNT,
-                 batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL):
+                 batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL,
+                 async_retry_options=ASYNC_RETRY_OPTIONS,
```

Comment:
I don't get used to use so many args in a constructor, it scares me a little bit:)
But well, yes, it should be easier for a finite user.